code/modules/ghostroles/spawner/simplemob/borer.dm

```diff
@@ -0,0 +1,28 @@
+/datum/ghostspawner/simplemob/borer
+	short_name = "borer"
+	name = "Cortical Borer"
+	desc = "Infest crew, reproduce, repeat."
+	tags = list("Antagonist")
+
+	spawn_mob = /mob/living/simple_animal/borer
+
+/datum/ghostspawner/simplemob/borer/select_spawnpoint(var/use)
+	return TRUE //We just fake it here, since the spawnpoint is selected if someone is spawned in.
+
+//The proc to actually spawn in the user
+/datum/ghostspawner/simplemob/borer/spawn_mob(mob/user)
+	var/mob/living/simple_animal/borer/spawn_borer
+	var/list/borer_list = list()
+
+	for(var/mob/living/simple_animal/borer/eligible_borers in SSghostroles.get_spawn_atom("Borers"))
+		borer_list += eligible_borers
+
+	if(!length(borer_list))
+		to_chat(user, SPAN_DANGER("There are no available borers to spawn at!"))
+		return FALSE
```

Comment:
Isn't this gonna have the same problem as golems where the spawner would permanently be up?

Reply:
i fixed that a long while a go with the disable enable thing, didn't i?

Reply:
Yeah but I don't see this spawner ever being closed until a borer dies. Why not do that after spawn_mob?

Reply:
![image](https://user-images.githubusercontent.com/22774890/79633320-f5df8280-8164-11ea-8cf4-78f4e6200980.png)

Reply:
Just tested it on runtime, it disables when I spawn into one spawned via the spawn menu. Working.

Reply:
For interest's sake, golem runes also work, just tested it.

Reply:
![image](https://user-images.githubusercontent.com/22774890/79633372-64244500-8165-11ea-8016-67a9f688924b.png)